compiler/rustc_ast/src/visit.rs

```diff
@@ -52,15 +53,15 @@ pub enum BoundKind {
 
     /// Super traits of a trait.
     /// E.g., `trait A: B`
-    SuperTraits,
+    SuperTraits { subtrait: LocalDefId },
```

Comment:
`LocalDefId` shouldn't be a part of AST or AST visitors.
If it's needed by some specific visitor, it's better to pass it in some other way.